apps/src/studio/studio.js

```diff
@@ -5353,6 +5353,11 @@ Studio.queueCallback = function (callback, args) {
   };
 
   registerEventHandler(Studio.eventHandlers, handlerName, () => {
+    // remove the last argument because stepCallExpression always wants to push it back on.
+    if (state.arguments.length > 0) {
+      state.value = state.arguments.pop();
+    }
+
     const depth = Studio.interpreter.stateStack.unshift(state);
```

Comment:
Dang. I thought JSInterpreter was the only thing that monkeyed with interpreter internals. This next line is going to be broken too with the PR that I have out right now but which is not merged yet. I guess I will try to write tests for this and/or factor this stuff out of `studio.js`